org.kde.alligator.json

```
@@ -0,0 +1,34 @@
+{
+    "id": "org.kde.alligator",
+    "rename-icon": "alligator",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "command": "alligator",
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri"
+    ],
+    "separate-locales": false,
```

```suggestion
    "separate-locales": false,
```
is this really needed?